### PR TITLE
Metachaser24: Initial Docs 

### PR DESCRIPTION
**Pull request by Divide-By-0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,423 +1,65 @@
...
-
-Note that local proving only works on Chrome/Brave/Arc (or other Chromium-based browsers) due to 1GB download limits on other browsers.
-
-While it is an open source public goods project, we intend to be thorough and deploy our SDK as well as applications built atop it into production. This is an always-evolving work in progress, and we invite folks to contribute, or contact us for interesting projects that can be built on top of the tech! We are especially prioritizing optimizing circuits, making more production level demos, and improved SDK/CLI tooling.
-
-### Credits
-This was originally conceived of in August 2022 by [@yush_g](https://twitter.com/yush_g) and [@sampriti0](https://twitter.com/sampriti0) at [@0xparc](https://twitter.com/0xparc), and has been improved since by several contributors supported by EF PSE including Sora, Saleel, Tyler, Sambhav, Javier, Rasul, Vivek, and more. We are grateful to circomlib, 0xparc, and double blind for their circuits. [DM us](http://t.me/zkemail) if interested in discussing usage, integrations, or building next generation primitives like this!
```
Comment (Divide-By-0) [inline README.md:22]: can you move these to the bottom

File: README.md [hunk 2]
```diff
@@ -1,423 +1,65 @@
...
-```
-const word_char = '(a|b|c|d|e|f|g|h|i|j|k|l|m|n|o|p|q|r|s|t|u|v|w|x|y|z|A|B|C|D|E|F|G|H|I|J|K|L|M|N|O|P|Q|R|S|T|U|V|W|X|Y|Z|0|1|2|3|4|5|6|7|8|9|_)';
-let regex = `email was meant for @${word_char}+`;
-```
+## Projects 🛠
+Take a look at all of the cool innovative projects that are building on top of Zk-Email!
 
-To understand these better, use https://cyberzhg.github.io/toolbox/ and use the 3 regex tools for visualization of the min-DFA state.
+- Zkp2p
+- Zk Email Safe
```
Comment (Divide-By-0) [inline README.md:60]: can you add links to these

**Comment by Divide-By-0** [comment]
are u able to deploy to render.com or have a specific build command i can put there?

**Comment by Divide-By-0** [comment]
Run git rebase main

**Comment by Metachaser24** [comment]
hey @isidroamv can you please fix my run_unit_and_e2e_tests errors. I am trying to merge and this keeps failing

**Comment by Divide-By-0** [comment]
have u fully rebased main?

These still seem to conflict:
package.json
packages/twitter-verifier-contracts/src/test/TestTwitter.t.sol
yarn.lock

**Comment by Metachaser24** [comment]
@Divide-By-0 I have. This afternoon i went through the whole process

**Comment by Divide-By-0** [comment]
Ok this is getting ridiculous lol. theres no reason any of these other random files should change rn; can we just isolate out the docs again into just docs/, get that merged, then figure the rest of these moves out?

**Pull request closed without merging**